docs/server/push-duty.rst

```diff
@@ -44,8 +44,14 @@ production should be cherry-picked::
 
     $ git checkout 2015.09.03  # Previous tag pushed to production.
```

Comment:
I guess this comment is stale now? It implies that any existing tag exists because it has been pushed to production which contradicts what you are proposing.